client/ayon_maya/plugins/load/load_arnold_standin.py

```diff
@@ -99,6 +99,7 @@ def load(self, context, name, namespace, options):
             cmds.setAttr(standin_shape + ".dso", path, type="string")
             sequence = is_sequence(os.listdir(os.path.dirname(repre_path)))
             cmds.setAttr(standin_shape + ".useFrameExtension", sequence)
+            cmds.setAttr(standin_shape + ".aiNamespace", namespace)
```

Comment:
This won't work. This will error I suspect. ;)

This needs `type="string"`, right?
```suggestion
            cmds.setAttr(standin_shape + ".aiNamespace", namespace, type="string")
```